docs/node/failures.md

```diff
@@ -64,28 +64,34 @@ As explained above, cancellation might not be the immediate cause of failure—i
 
 ```ts
 import {
-  Context,
   CancellationScope,
+  configureActivities,
   isCancellation,
 } from '@temporalio/workflow';
 import * as activities from '../activities';
 
-const { httpGetJSON } = Context.configureActivities<typeof activities>({
-  type: 'remote',
-  startToCloseTimeout: '1m',
-});
-
-export async function main(urls: string[], timeoutMs: number): Promise<any[]> {
-  try {
-    return CancellationScope.withTimeout(timeoutMs, () =>
-      Promise.all(urls.map((url) => httpGetJSON(url)))
-    );
-  } catch (err) {
-    if (isCancellation(err)) {
-      console.log('Deadline exceeded while waiting for activities to complete');
-    }
-    throw err;
-  }
+export function myWorkflow(urls: string[], timeoutMs: number) {
+  const { httpGetJSON } = configureActivities<typeof activities>({
+    type: 'remote',
```

Comment:
what is `type`?

Reply:
https://nodejs.temporal.io/api/namespaces/worker#activityoptions

I imagine remote is the default?

Reply:
I like an optional boolean w/ default false

Reply:
`local: false` is fine until we need to add a third type of activity, i'm not sure if we ever will so i guess its ok

i dislike `type: 'remote'` because "remote activity" is not defined anywhere else in other SDKs.